src/NewsService.Mappers/Models/Interfaces/INewsInfoMapper.cs

```diff
@@ -2,14 +2,11 @@
 using LT.DigitalOffice.NewsService.Mappers.Interfaces;
 using LT.DigitalOffice.NewsService.Models.Db;
 using LT.DigitalOffice.NewsService.Models.Dto.Models;
-using System;
-using System.Collections.Generic;
 
 namespace LT.DigitalOffice.NewsService.Mappers.Models.Interfaces
 {
-    [AutoInject]
-    public interface IDbNewsMapper
-    {
-        DbNews Map(News request, List<Guid> departmentId);
-    }
+  [AutoInject]
+  public interface INewsInfoMapper : IMapper<DbNews, NewsInfo>
```

Comment:
at one time, we abandoned the use of this interface

Reply:
We discussed with @MariyaSin , decided what was needed